fimsauthz-api/utm-roles.yaml

```diff
@@ -64,19 +65,6 @@ VEHICLE_REGISTRATION_CHECKER:
       - "USS queries a vehicle data provider using its uvin scopes to check whethter a UVIN is registered."
       - "A Public Safety USS queries a vehicle data provider using its uvin scopes to learn the vehicle's owner."
 
-CONFLICT_MANAGEMENT:
```

Comment:
@issmith1 we are losing this role since ever USS should have it?  Is that correct?